.github/workflows/monthly-warning-test.yml

```diff
@@ -0,0 +1,58 @@
+name: Test for Warnings
+
+on:
+  workflow_dispatch: null
+  schedule:
+    # Runs "First of every month at 3:15am Central"
+    - cron: '15 8 1 * *'
+  push:
```

Comment:
@evevkovacs these last few lines starting with `push` should be removed once we have a passing workflow